src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs

```diff
@@ -25,7 +25,8 @@ public static class ResNet18Extension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> ResNet18(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return ResNet18(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "DnnImageModels"));
+            string fullPath = new DirectoryInfo(System.Reflection.Assembly.GetAssembly(typeof(DnnImageModelSelector)).Location).Parent.FullName;
```

Comment:
Why are we just changing ResNet18, and not the other DnnImageFeaturizers?

Reply:
Just lazy.... I will change the other two after you think the change is necessary...